showcases/react-showcase/src/components/default-component.tsx

```diff
@@ -115,27 +115,8 @@ const DefaultComponent = ({
 	const color = useQuery(redirectURLSearchParameters)[2];
 
 	const getHref = (variantName: string): string => {
-		if (typeof globalThis !== 'undefined') {
-			const searchParameters = new URLSearchParams(
-				globalThis?.location?.href.split('?')[1]
-			);
-			searchParameters.set('page', variantName.toLowerCase());
-			return `${globalThis?.location?.href.split('?')[0]}?${searchParameters.toString()}`;
-		}
-
-		return '';
-	};
-
-	const openVariantInNewWindow = (
-		event: React.MouseEvent<HTMLAnchorElement>,
-		variantName: string
-	) => {
-		if (
-			typeof globalThis === 'undefined' ||
-			!globalThis.location.origin ||
-			!globalThis.location.href
-		) {
-			return;
+		if (typeof globalThis === 'undefined' || !globalThis.location.href) {
+			return '';
 		}
 
 		const currentUrl = globalThis.location.href.split('?');
```

Comment:
```suggestion
		const [baseUrl, query=''] = globalThis.location.href.split('?');
```
We could use it the following lines like this instead of using the index:

`const searchParameters = new URLSearchParams(query);`